ignite/services/network/networkchain/init.go

```diff
@@ -101,7 +104,39 @@ func (c *Chain) initGenesis(ctx context.Context) error {
 		if err := os.WriteFile(genesisPath, genBytes, 0o644); err != nil {
 			return err
 		}
-	} else {
+
+	case c.genesisConfig != "":
+		c.ev.Send("Fetching custom Genesis from Config", events.ProgressStarted())
+
+		// find config in downloaded source
+		path := filepath.Join(c.path, c.genesisConfig)
+		if _, err := os.Stat(path); err != nil {
+			return err
```

Comment:
Could we wrap the error to say: `the config for genesis doesn't exist`
Otherwise the output is
```
Source code fetched
Blockchain set up
Chain's binary built
Blockchain initialized
stat /var/folders/6n/c3hb98tx2sjdxbz0cg5fy2gm0000gn/T/3438528053/test.yml: no such file or directory
```